### PR TITLE
decommission stack images

### PR DESCRIPTION
**Pull request by shubham805**

### CODE REVIEW

File: GenerateDockerFiles/dockerFilesGenerateTask.yml [hunk 1]
```diff
@@ -29,62 +29,6 @@ steps:
...
-    args: $(Build.ArtifactStagingDirectory) ${{ parameters.appSvcGitUrl }} $(System.DefaultWorkingDirectory)/Config
-
-- task: ShellScript@2
-  displayName: 'Generate Wordpress Docker Files'
-  condition: or(eq('${{ parameters.stack }}', ''), eq('${{ parameters.stack }}','wordpress'))
-  inputs:
-    scriptPath: ./GenerateDockerFiles/wordpress/generateDockerfiles.sh
-    args: $(Build.ArtifactStagingDirectory) $(System.DefaultWorkingDirectory)/Config
-
 - task: ShellScript@2
```
Comment (vijaysaayi) [inline GenerateDockerFiles/dockerFilesGenerateTask.yml:32]: Kudulite test case project depends on dotnetcore and node runtime images. Could you check this
[ImageBuilder/HelloWorld.cs at master · Azure-App-Service/ImageBuilder (github.com)](https://github.com/Azure-App-Service/ImageBuilder/blob/master/Tests/KuduLite/HelloWorld.cs)

**Comment by surenderssm** [comment]
Can we add some details as part of PR description.

File: BuildAndTagImages/build.sh [hunk 2]
```diff
@@ -55,7 +55,7 @@ function buildDockerImage()
     if [ -f "$CONFIG_DIR/${STACK}VersionTemplateMap.txt" ]; then
         while IFS=, read -r STACK_VERSION BASE_IMAGE STACK_VERSION_TEMPLATE_DIR STACK_TAGS || [[ -n $STACK_VERSION ]] || [[ -n $BASE_IMAGE ]] || [[ -n $STACK_VERSION_TEMPLATE_DIR ]] || [[ -n $STACK_TAGS ]]
         do
-            if [[ -z $STACK_VERSION_TO_BUILD || $STACK_VERSION_TO_BUILD == $STACK_VERSION ]];
+            if [[ -z $STACK_VERSION_TO_BUILD || $STACK_VERSION_TO_BUILD == $STACK_VERSION]];
```
Comment (surenderssm) [inline BuildAndTagImages/build.sh:58]: What is the change here, can we undo the space issue as well

File: BuildAndTagImages/build.sh [hunk 3]
```diff
@@ -111,7 +111,7 @@ function buildDockerImage()
                         docker build -t "$BuildVerRepoTag" -f "$appSvcDockerfilePath" .
                     fi
 
-                    if [ "$BUILD_REASON" != "PullRequest" ]; then
+                    if [ "$BUILD_REASON" != "PullRequest" && $STACK = "wordpress"]; then
```
Comment (surenderssm) [inline BuildAndTagImages/build.sh:114]: Will this just enable WordPress to be built with this?
How about kudulite and diagnostic server?
Can we add a comment explaining the reasoning behind this change for future reference.

**Pull request closed without merging**